python/ray/serve/config.py

```diff
@@ -30,25 +30,27 @@ class BackendMetadata:
 class BackendConfig(BaseModel):
     """Configuration options for a backend, to be set by the user.
 
-    :param num_replicas: The number of processes to start up that will
-        handle requests to this backend. Defaults to 0.
-    :type num_replicas: int, optional
-    :param max_batch_size: The maximum number of requests that will be
-        processed in one batch by this backend. Defaults to None (no
-        maximium).
-    :type max_batch_size: int, optional
-    :param batch_wait_timeout: The time in seconds that backend replicas will
-        wait for a full batch of requests before processing a partial batch.
-        Defaults to 0.
-    :type batch_wait_timeout: float, optional
-    :param max_concurrent_queries: The maximum number of queries that will be
-        sent to a replica of this backend without receiving a response.
-        Defaults to None (no maximum).
-    :type max_concurrent_queries: int, optional
-    :param user_config: Arguments to pass to the reconfigure method of the
-        backend. The reconfigure method is called if user_config is not
-        None.
-    :type user_config: Any, optional
+    Args:
+        num_replicas (Optional[int]): The number of processes to start up that
+            will handle requests to this backend. Defaults to 0.
+        max_batch_size (Optional[int]): The maximum number of requests that
+            will be processed in one batch by this backend. Defaults to None
+            (no maximium).
+        batch_wait_timeout (Optional[float]): The time in seconds that backend
+            replicas will wait for a full batch of requests before processing a
+            partial batch. Defaults to 0.
+        max_concurrent_queries (Optional[int]): The maximum number of queries
+            that will be sent to a replica of this backend without receiving a
+            response. Defaults to None (no maximum).
+        user_config (Optional[Any]): Arguments to pass to the reconfigure
+            method of the backend. The reconfigure method is called if
+            user_config is not None.
+        experimental_graceful_shutdown_wait_loop_s (Optional[float]): Duration
+            that backend workers will wait until there is no more work to be
+            done before shutting down. Defaults to 2s.
```

Comment:
Maybe mention that  `>= time_for_router_long_polling_update`

Reply:
hmm this value isn't expected to be updated, it should be set by create_backend and populated on replica construction.